app/templates/direct-award/view-project.html

```diff
@@ -135,7 +135,7 @@
           }
         ] %}
           <li class="instruction-list-item divider">
-            <h2 class="instruction-list-item-body"><span class="step-number" role="presentation">{{ loop.index }}.</span>&nbsp;{{ step.title }}</h2>
+            <h2 class="instruction-list-item-body"><span class="step-number" role="presentation">{{ loop.index }}.</span>{{ step.title }}</h2>
```

Comment:
I think whatever the CSS, there ought to be at least one space (of some sort) in the actual markup. For accessibility if nothing else.

Reply:
(I don't actually see why we expect anything other than the numbers to line up on the left. There shouldn't be a requirement for the text after the numbers to line up.)

Reply:
We needed to lose the space from the step.title to get it aligned properly with the paragraph below.

We could add a space after the loop.index, but it isn't required. VoiceOver, etc will handle reading it out normally and I can't think of any other accessibility issues it might cause.

Reply:
hmm, I think the space is required for this to be correct English, but I'm willing to believe it makes little practical difference